pkg/gadgets/trace/mount/types/types.go

```diff
@@ -15,27 +15,57 @@
 package types
 
 import (
+	"fmt"
+	"strings"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64   `json:"mntnsid,omitempty"`
-	Pid       uint32   `json:"pid,omitempty"`
-	Tid       uint32   `json:"tid,omitempty"`
-	Comm      string   `json:"comm,omitempty"`
-	Operation string   `json:"operation,omitempty"`
-	Retval    int      `json:"ret,omitempty"`
-	Latency   uint64   `json:"latency,omitempty"`
-	Fs        string   `json:"fs,omitempty"`
-	Source    string   `json:"source,omitempty"`
-	Target    string   `json:"target,omitempty"`
-	Data      string   `json:"data,omitempty"`
-	Flags     []string `json:"flags,omitempty"`
+	Comm      string   `json:"comm,omitempty" column:"comm,maxWidth:16"`
+	Pid       uint32   `json:"pid,omitempty" column:"pid,minWidth:7"`
+	Tid       uint32   `json:"tid,omitempty" column:"tid,minWidth:7"`
+	MountNsID uint64   `json:"mntnsid,omitempty" column:"mnt_ns,width:12"`
```

Comment:
We are using `mntns` for other gadgets:

```suggestion
	MountNsID uint64   `json:"mntnsid,omitempty" column:"mntns,width:12"`
```

I think this changes implies you will have to update the [documentation](https://github.com/kinvolk/inspektor-gadget/blob/main/docs/guides/trace/mount.md).